README.md

```diff
@@ -22,6 +22,8 @@ Here you will find a growing repository of documentation for the Editorial Team.
 
 Take a look at [our contribution guide](CONTRIBUTING.md#joining-the-hoodie-editorial-team). We're looking for people of all sorts, so don't be afraid to get involved. If you're passionate about contributing to a great community, we want you! :yellow_heart:
 
+If you have anything you'd like to contribute, please fork this repository and send a Pull Request for your contribution. Hoodie has [a great article](http://hood.ie/blog/contributing-to-hoodie.html) that explains how to fork and edit files. It's especially useful if you're new to the process.  
```

Comment:
@nickcolley I worked through it with @Charlotteis as I was actively writing it, and @gr2m reviewed it as I was waiting for my merge permissions to become active.